test/index.test.js

```diff
@@ -102,166 +105,252 @@ describe('AdRoll', function() {
         analytics.stub(window.__adroll, 'record_user');
       });
 
-      it('should track page view with fullName', function() {
-        analytics.page('Category', 'Name', { url: 'http://localhost:34448/test/' });
-        analytics.called(window.__adroll.record_user, {
-        adroll_segments: 'viewed_category_name_page',
-        adroll_custom_data: {
-          path: '/test/',
-          referrer: '',
-          title: 'integrations tests',
-          search: '',
-          name: 'Name',
-          category: 'Category',
-          url: 'http://localhost:34448/test/'
-        }
+      describe('V1', function() {
```

Comment:
Any file in the `test/` directory named `*.test.js` will get picked up by the test runner.